pkg/bootstrapper/service.go

```diff
@@ -0,0 +1,258 @@
+package bootstrapper
+
+import (
+	"fmt"
+	"time"
+
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+// kubeletService struct contains the kubelet specific service information
+type kubeletService struct {
```

Comment:
Technically you specialized here but the fields are generic enough. However there is no problem with this approach considering we just have a single service for now.